gst-libs/mfx/gstmfxprimebufferproxy.c

```diff
@@ -199,15 +220,15 @@ gst_mfx_prime_buffer_proxy_get_handle (GstMfxPrimeBufferProxy * proxy)
 {
   g_return_val_if_fail (proxy != NULL, 0);
 
-  return proxy->fd;
+  return proxy->buf_info.handle;
 }
 
 guint
 gst_mfx_prime_buffer_proxy_get_size (GstMfxPrimeBufferProxy * proxy)
 {
   g_return_val_if_fail (proxy != NULL, 0);
 
-  return proxy->data_size;
+  return proxy->buf_info.mem_size;
```

Comment:
proxy->data_size is already assigned, so this change is not necessary.

Reply:
I think it makes more sense for fd to be assigned to the buf_info structure instead of the proxy structure. proxy size/handle implies the size and handle of the buffer proxy itself not the actual buffer that the buffer proxy is suppose to carry. This to me makes more logical sense.

Reply:
Refer to my last previous comment related to this change.